### ISSUE TITLE
Lenses not available any more ?

### ISSUE THREAD

**Issue by vospascal** [relativty/Relativty#59]
No store behind the lens might be an idea to make a list of stores And places where you can source the components not just one link for every component

**Comment by ezar** [comment]
Same here!

**Comment by okawo80085** [comment]
I mean you can make this list yourself if you want  to

**Comment by isaaclepes** [comment]
Yeah, it's hard to find a seller for 40mm dia. 50mm focal len. lenses.. all of them seem to be 42mm.  Since the focal length is what's most important, just get as close to the diameter as you can and modify the 3d model to hold the new diameter lenses.
